tools/pd-ctl/pdctl/command/keyspace_command.go

```diff
@@ -317,3 +321,119 @@ func listKeyspaceCommandFunc(cmd *cobra.Command, args []string) {
 	}
 	cmd.Println(resp)
 }
+
+func newShowKeyspaceRangeCommand() *cobra.Command {
+	r := &cobra.Command{
+		Use:   "range",
+		Short: "show keyspace range information",
+	}
+	r.PersistentFlags().Bool("raw", false, "show raw key range (default: txn key range)")
+	rangeByID := &cobra.Command{
+		Use:   "id <keyspace_id>",
+		Short: "show key ranges for the given keyspace id",
+		Run:   showKeyspaceRangeByIDCommandFunc,
+	}
+
+	rangeByName := &cobra.Command{
+		Use:   "name <keyspace_name>",
+		Short: "show key ranges for the given keyspace name",
+		Run:   showKeyspaceRangeByNameCommandFunc,
+	}
+	r.AddCommand(rangeByID)
+	r.AddCommand(rangeByName)
+	return r
+}
+
+func showKeyspaceRangeByIDCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 1 {
+		cmd.Usage()
+		return
+	}
+
+	keyspaceID := args[0]
```

Comment:
Do we need check id exists?

Reply:
I think it would be a good idea to check it out, it doesn't cost any resources

Reply:
No, it's just a kind of way to get the key range, like the tso command.